pkgs/top-level/all-packages.nix

```diff
@@ -4571,8 +4571,8 @@ with pkgs;
 
   dotnetfx40 = callPackage ../development/libraries/dotnetfx40 { };
 
-  dolphinEmu = callPackage ../misc/emulators/dolphin-emu { };
-  dolphinEmuMaster = qt5.callPackage ../misc/emulators/dolphin-emu/master.nix {
```

Comment:
Please add aliases to the old names on `pkgs/top-level/aliases.nix` to avoid breaking old configurations.